README.md

```diff
@@ -108,62 +114,6 @@ The following example details how to embed the API Designer:
 </html>
 ```
 
-## Development
```

Comment:
I'd replace this section with a link to the contributing docs instead of deleting entirely.